.ci/ci.yml.tmpl

```diff
@@ -113,8 +119,28 @@ resources:
         uri: git@github.com:((github-account.username))/puppet-google-{{module}}.git
 {% endif %}
         private_key: ((repo-key.private_key))
+
+{% if module != "bigquery" %}
```

Comment:
What's special about bigquery? Presumably that it's not done yet; maybe add a TODO or file a bug to remember to enable it here and below once it's done.

Reply:
Or should there be a different means of excluding in-progress products, so we don't accidentally publish incomplete future products? (asked by someone who knows nothing about Puppet Forge)

Reply:
Very much in agreement.  Added that, also refactored the whole deal so it's separate files and is deployed to separate pipelines.  That will be a big help for keeping all this manageable.